docs/clean.rst

```diff
@@ -353,7 +353,7 @@ Trivial Filter example:
 .. doctest::
 
    >>> from bleach.sanitizer import Cleaner
-   >>> from html5lib.filters.base import Filter
+   >>> from bleach.html5lib_shim import Filter
```

Comment:
This one is interesting. What's going on is that we vendored `html5lib`, so it's no longer installed as a requirement. That's cool, but that means it's not installed anymore. I switched it to use the `bleach.html5lib_shim` module. It's the same `Filter` class, but it's specifically the one that ships with Bleach. That's more correct. I'm not sure if it feels weird to be importing it from a module named `bleach.html5lib_shim` or not. It's probably fine.

Reply:
wfm